.ci/update-references.py

```diff
@@ -0,0 +1,17 @@
+import json
+import os
+import sys
+
+REF = sys.argv[1]
+
+
+for file in os.listdir("examples"):
+    recipe_path = os.path.join("examples", file, ".saturn/saturn.json")
+
+    with open(recipe_path) as f:
+        recipe = json.load(f)
+        for repo_dict in recipe["git_repositories"]:
+            repo_dict["reference"] = REF
```

Comment:
Will this require us to always include references in the recipes (as in we must include it even if it's `main`)? I think I am okay with that tradeoff I just want to make sure I understand correctly

Reply:
no it doesn't imply that. This just makes sure that we _do_ have references on release branches.